WalkingChallenge/DropDownPicker.swift

```diff
@@ -0,0 +1,195 @@
+
+import UIKit
+
+class DropDownPicker: UIControl {
+  fileprivate var picker: UIPickerView!
+  fileprivate var toolbar: UIToolbar!
+
+  var textfield: UITextField!
+
+  var data: Array<String>?
+
+  var placeholder: String = "Tap to choose ..." {
```

Comment:
I think that we can do that external to this file, so we can tweak it for the different views.